Assets/Ekaterina/Growth/Scripts/Win_game.cs

```diff
@@ -16,19 +16,22 @@ public class Win_game : MonoBehaviour
         public Button Button;
         
         void Start () {
-            Button btn = Button.GetComponent<Button>();
-            btn.onClick.AddListener(TaskOnClick);
+            if (SceneManager.GetActiveScene().name == "Ekaterina.Growth.Win")
+            {
+                Button btn = Button.GetComponent<Button>();
+                btn.onClick.AddListener(TaskOnClick);
+            }
         }
 
         void TaskOnClick(){
-            SceneManager.LoadScene("Main Scene 1");
+            SceneManager.LoadScene("Ekaterina.Growth.Main scene 1");
```

Comment:
Please take some time to review this set of changes! I made it so that the code reacts based on the scene and also gave the scenes more specific names.